### ISSUE TITLE
linter: eslint-plugin-jsx-a11y(label-has-associated-control)

### ISSUE THREAD

**Issue by coreAux** [oxc-project/oxc#11644]
### What version of Oxlint are you using?

1.1.0

### What command did you run?

oxlint

### What does your `.oxlintrc.json` config file look like?

```jsonc
{
  "plugins": [
    "eslint",
    "import",
    "jsx-a11y",
    "nextjs",
    "node",
    "oxc",
    "promise",
    "react",
    "react-perf",
    "typescript",
    "unicorn"
  ],
  "rules": {
    "import/no-default-export": 0,
    "no-console": 1,
    "react/button-has-type": 0,
    "react/rules-of-hooks": 2,
    "typescript/explicit-function-return-type": 0,
    "typescript/no-non-null-assertion": 0,
    "unicorn/filename-case": [2, { "case": "kebabCase" }],
    "unicorn/no-nested-ternary": 0
  }
}

```

### What happened?

Linter is not detecting that my `<label>`-tags are wrapped around the input control, see screenshot.

```tsx
<label>
  <span>name</span>
  <input
    defaultValue={
      actionState.payload?.get("name")?.toString() ?? "Pool party"
    }
    name="name"
    placeholder="name"
    type="text"
  />
</label>
```

![Image](https://github.com/user-attachments/assets/622bbb90-94b5-4dcd-9de9-dba694676b94)
